src/smc-webapp/editor.coffee

```diff
@@ -1075,6 +1075,14 @@ class exports.Editor
             if @open_file_tabs().length < 1
                 @resize_open_file_tabs()
 
```

Comment:
What's the block of code below supposed to do?   I couldn't understand it and there are no comments.